Changelog.md

```diff
@@ -23,6 +23,11 @@
 - [#490]: Removed `$unflatten=` special prefix for fields for serde (de)serializer, because:
   - it is useless for deserializer
   - serializer was rewritten and does not require it anymore
+- [#490]: Special name `$value` for a field has been replaced by two special names:
+  - `#text` if you want to map field to text content only. No markup is expected
+    (but text can represent a list as defined by `xs:list` type)
+  - `#any` that is close to the previous `$value` behavior. Use it when you
+    want to map field to any content of the field, text or markup
```

Comment:
An example might also be nice - or a link to documentation where you can read more.

Reply:
Added link to documentation